ete2/parser/newick.py

```diff
@@ -237,6 +242,21 @@ class as the root(This allows to work with custom TreeNode
 
 def _read_newick_from_string(nw, root_node, format):
     """ Reads a newick string in the New Hampshire format. """
+    # begin: Quoted text is mapped to references
+    quoteMap={}
+    unquoted_nw=''
+    counter=0
+    for token in re.split(_QUOTED_TEXT_RE, nw):
+        counter+=1
+        if counter % 2 == 1 : #normal newick tree structure data
```

Comment:
What is a "normal newick tree structure" ?
I don't know what this comment means.